### PR TITLE
Fix issue 24987: introduce template literals

### LINKED ISSUE #24987: A first splash into JavaScript does not explain template literals before use
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/A_first_splash

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

This code was written in the article _A first splash into JavaScript_
``guesses.textContent += `${userGuess} `;``
In the complete code on another page the above line is replaced by:
`guesses.textContent += userGuess + ' ';`
which makes the effect of the earlier line here apparent but doesn't give a clear explanation of its syntax.

The article is aimed at newcomers to JavaScript, however there is no previous reference to the meaning of `${}` and a Google search provides no help.   

### What did you expect to see?

More detail

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/first_steps/a_first_splash`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/A_first_splash
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/first_steps/a_first_splash/index.md
* Last commit: https://github.com/mdn/content/commit/7ff26ceb6c6f1400d4297b22cb92e3786e35eac7
* Document last modified: 2023-02-26T18:57:36.000Z

</details>

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/24987:

- stop talking about using `+` for string concatenation
- introduce template literals - though I never know how much to go into details in super-intro material like this
- update the code so it consistently uses template literals for concatenation

We ought to update https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/first-splash/number-guessing-game.html as well - it's already divergent from the code in the page 😭 .

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/a_first_splash/index.md [hunk 1]
```diff
@@ -183,29 +183,18 @@ First let's look at arithmetic operators, for example:
 | `*`      | Multiplication | `3 * 7`   |
 | `/`      | Division       | `10 / 5`  |
 
-You can also use the `+` operator to join text strings together (in programming, this is called _concatenation_). Try entering the following lines, one at a time:
+There are also some shortcut operators available, called augmented [assignment operators](/en-US/docs/Web/JavaScript/Reference/Operators#assignment_operators). For example, if you want to add a new number to an existing one and return the result, you could do this:
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/first_steps/a_first_splash/index.md:186]:
```suggestion
There are also some shortcut operators available, called [compound assignment operators](/en-US/docs/Web/JavaScript/Reference/Operators#assignment_operators). For example, if you want to add a new number to an existing one and return the result, you could do this:
```

File: files/en-us/learn/javascript/first_steps/a_first_splash/index.md [hunk 2]
```diff
@@ -278,7 +289,7 @@ function checkGuess() {
   if (guessCount === 1) {
     guesses.textContent = "Previous guesses: ";
   }
-  guesses.textContent += `${userGuess} `;
+  guesses.textContent = `${guesses.textContent}${userGuess} `;
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/first_steps/a_first_splash/index.md:292]: I reckon this is better:

```js
  if (guessCount === 1) {
    guesses.textContent = "Previous guesses:";
  }
  guesses.textContent = `${guesses.textContent} ${userGuess}`;
```

File: files/en-us/learn/javascript/first_steps/a_first_splash/index.md [hunk 3]
```diff
@@ -316,7 +327,7 @@ This is a lot of code — phew! Let's go through each section and explain what i
 
   If it is, we make the guesses paragraph's text content equal to `Previous guesses:`. If not, we don't.
 
-- Line 6 appends the current `userGuess` value onto the end of the `guesses` paragraph, plus a blank space so there will be a space between each guess shown.
+- Next, we use a template literal to append the current `userGuess` value onto the end of the `guesses` paragraph, plus a blank space so there will be a space between each guess shown.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/first_steps/a_first_splash/index.md:330]: If using the improvement above, this sentence should be changed.

**Review by Josh-Cena — COMMENTED** [review]
Looking at https://github.com/mdn/content/issues/28545, should we update that page as well, since that one's specifically about strings?

**Comment by wbamberg** [comment]
> Looking at #28545, should we update that page as well, since that one's specifically about strings?

Yes, I was going to try to address these together, but once I was in it, it seemed better to do them in separate PRs.

**Review by Josh-Cena — APPROVED** [review]
LGTM :)

**Pull request merged**